misc/history-service-specific.json

```diff
@@ -3509,79 +3509,79 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sum(rate(schedule_activity_command{service_name=\"history\", operation=\"RespondWorkflowTaskCompleted\"}[5m]))",
+          "expr": "sum(rate(command{service_name=\"history\", operation=\"RespondWorkflowTaskCompleted\", commandType=\"ScheduleActivityTask\"}[5m]))",
```

Comment:
instead of writing them in separate query, can we do one query and sum by commandType?